packages/edit-site/src/components/block-editor/index.js

```diff
@@ -153,6 +153,8 @@ export default function BlockEditor( { setIsInserterOpen } ) {
 		// Disable resizing in mobile viewport.
 		! isMobileViewport;
 	const isViewMode = canvasMode === 'view';
+	const showBlockAppender =
+		( isTemplatePart && hasBlocks ) || isViewMode ? false : undefined;
```

Comment:
This change will hide the appender at the top level, do we have a similar issue for inner appenders?

Reply:
Maybe be we don't because we unselect all blocks, but it would be good to confirm that we won't see these for any block.

If that's the case, we can ship this otherwise we'll have to figure out a different way to fix it (maybe inject CSS in the iframe)

Reply:
I tried several patterns and any inserters didn't show up in view mode. However, it seems that it is already well controlled on `trunk`, except for the root level inserter that this PR fixes.

https://user-images.githubusercontent.com/54422211/209354010-e3811a3b-5ebe-4d67-846a-4eb0f04d9a46.mp4